src/controllers/users.ts

```diff
@@ -21,3 +23,24 @@ export function getCats(request: Request, response: Response) {
 		});
 	});
 }
+
+export function updateUser(req: Request, res: Response, next: NextFunction) {
+	const schema = {
+		requested_privacy: 'boolean',
+	};
+	const payload = req.body;
+	const result = validator(payload, schema);
+
+	if (!result.success) return next({ status: 400, message: result.errors });
+
+	const id: string = req.headers.authorization ?? '';
```

Comment:
You can just use the middleware I made :) Takes care of everything

Reply:
How?